src/txmempool.h

```diff
@@ -515,6 +515,8 @@ class CTxMemPool
     void PrioritiseTransaction(const uint256& hash, const CAmount& nFeeDelta);
     void ApplyDelta(const uint256& hash, CAmount &nFeeDelta) const EXCLUSIVE_LOCKS_REQUIRED(cs);
     void ClearPrioritisation(const uint256& hash) EXCLUSIVE_LOCKS_REQUIRED(cs);
+    /** Return a copy of mapDeltas in which values include fee delta and whether the transaction is present in the mempoool. */
```

Comment:
Slight rephrasing suggestion:
```suggestion
    /** Return a map of all entries in mapDeltas with as value their fee delta and whether the transaction is present in the mempoool. */
```

Reply:
Done